test/utils-tests.jl

```diff
@@ -468,6 +468,15 @@ let
     @test obl  ≈ [-9.507794266102866, -6.970768250588256]
 end
 
+# Test obliquity
+@testset "obliquity" begin
+    @test obliquity(2.451545e6) ≈ 0.4090646078966446
```

Comment:
If a symbol isn't exported, you can reference it by prefixing it with the module where it's defined (`AstroLib.J2000`) ;-)   J2000 is indeed a good test case, because the obliquity is known with precision in that case!